sphinx/ext/autosummary/generate.py

```diff
@@ -79,6 +80,11 @@ def _simple_info(msg):
 def _simple_warn(msg):
     print('WARNING: ' + msg, file=sys.stderr)
 
+def _underline(title, line='='):
+    if '\n' in title:
+        raise ValueError('Can only underline single lines')
+    return title + '\n' + line*len(title)
```

Comment:
Could you use `docutils.utils.column_width()` instead of `len()`?
It supports east asian width. They can use character having multiple widths for module name. So it's important for asian users.